docs/guide/multisite.md

```diff
@@ -2,109 +2,202 @@
 
 ## Introduction
 
-With eZ Platform it is possible to serve multiple, completely different sites using one eZ Platform instance and database.
+You can serve multiple different sites using one eZ Platform instance and database.
 
-Each site will have its own content root, at a lower level than the default one (Location ID 2). One backoffice can be used for each site, but it is also possible to use a global one.
+Each site will have its own content root, at a lower level than the default one (Location ID 2).
+You can use one global back office for all sites, or a separate back office for each site.
```

Comment:
Why do we use a two-words  `backoffice` now?

Reply:
Because [there](https://en.wiktionary.org/wiki/back_office) [is](http://www.dictionary.com/browse/back-office?s=t) [no reason](https://en.wikipedia.org/wiki/Front_and_back_office_application) to write it as one word ;)